tasks/process_agent.py

```diff
@@ -96,3 +98,42 @@ def build(ctx, race=False, go_version=None, incremental_build=False,
     }
 
     ctx.run(cmd.format(**args), env=env)
+
+
+@task
+def build_dev_image(ctx, image=None, push=False, base_image="datadog/agent:latest"):
+    """
+    Build a dev image of the process-agent based off an existing datadog-agent image
+
+    image: the image name used to tag the image
+    push: if true, run a docker push on the image
+    base_image: base the docker image off this already build image (default: datadog/agent:latest)
+    """
+    if image == None:
```

Comment:
nit: `if image is None` is the recommended approach (https://www.python.org/dev/peps/pep-0008/). Also, why do we need a default value of `None` if a value is mandatory?